### ISSUE TITLE
`over_booksize`:  Booksize number weight parameter

### ISSUE THREAD

**Issue by DandeVilliers** [TheProfitTable/loanportr#22]
Currently the function `over_booksize` calculates the booksize number for this month, last month
and the % change in value from last month against `net_advance`. To generalize this function more it might be weighted against `closing_balance` as well.

Change the function to take in an additional weight parameter that can incorporate this.

**Comment by catherineliu2102** [comment]
Hi,
I'm trying to install the package using 
devtools::install_github("TheProfitTable/loanportr")
but the error says:
Skipping install of 'loanportr' from a github remote, the SHA1 (51092581) has not changed since last install.
Use `force = TRUE` to force installation

Is there any thing wrong?

Thanks~

**Comment by DandeVilliers** [comment]
Hi @catherineliu2102,

Have you tried `devtools::install_github("TheProfitTable/loanportr", force=TRUE)`. Inculding the `force` parameter seems to do the trick.

Let me know if you experience any issues. Thanks

reference: https://stackoverflow.com/questions/39584717/using-devtools-github-install-without-force-true

**Comment by catherineliu2102** [comment]
Hi Dan,

Thanks for the quick response. Can you elaborate more on ' weighted against closing_balance as well' (assuming this is an issue that new kicks can work on)？

**Comment by DandeVilliers** [comment]
Yes, so basically if you look within the `over_booksize` function you'll see (at line 130 I believe) the `sum(net_advance)`. In other words `over_booksize` will always use `net_advance` as a weight parameter. 

We, however, need the function to take in other weight parameters such as `closing_balance`. So you need to replace `net_advance` within the function with a weight parameter let's call it `weight` and have that as an input parameter to the function. Make sure to test the function with `closing_balance` as well. Hope it makes sense.

Thanks, @catherineliu2102
